.github/workflows/build_binaries.yml

```diff
@@ -0,0 +1,66 @@
+name: Build Binaries # Continuous Deployment
```

Comment:
IIUC we don't need to build binary for library crates?

Reply:
Deleted the file, good point